.buildkite/pipeline.yml

```diff
@@ -137,11 +135,11 @@ steps:
       - weeder
     commands:
       - eval "$FETCH_CONTRACT"
-      - nix-build -A stablecoin-client -o stablecoin-static
-      - nix run -f . pkgs.upx -c upx stablecoin-static/bin/stablecoin-client -o stablecoin-client
+      - nix build .#stablecoin:exe:stablecoin-client -o stablecoin-static
```

Comment:
please use `-L` flag with every `nix build` invocation, except those where we use `--print-out-paths`